src/main/resources/project/project.xml

```diff
@@ -6,6 +6,83 @@
     <description>Integration with WebSphere</description>
     <workspaceName />
     <propertySheet>
+        <!-- CEV-17558 improvements -->
+        <property>
+            <propertyName>ec_config</propertyName>
+            <description/>
+            <propertySheet>
+                <!-- WebSphere doesn't have custom GWT logic. Setting it for plugin context -->
+                <property>
+                    <propertyName>ec_formXmlCompliant</propertyName>
```

Comment:
ec_formXmlCompliant property should be at the plugin level *not* within ec_config property sheet.

Reply:
Thanks, will fix it